### PR TITLE
Release v1.14.0

### PR DESCRIPTION
**Pull request by TomKimsey**

### CODE REVIEW

**Review by alexjhawk — APPROVED** [review]
I think this looks good to me. 

Also, what do you and @it-hms think about adding automation to help us better update connectors when the extensions library releases are made? I think it would be extremely useful to automatically log an issue on connector repositories that an update needs to be made, otherwise features like this tend to be forgotten and only make it to 1 or 2 connectors.

**Comment by it-hms** [comment]
> I think this looks good to me.
> 
> Also, what do you and @it-hms think about adding automation to help us better update connectors when the extensions library releases are made? I think it would be extremely useful to automatically log an issue on connector repositories that an update needs to be made, otherwise features like this tend to be forgotten and only make it to 1 or 2 connectors.

Yeah, it makes sense to have automation to log an issue.

File: src/main/java/com/hms_networks/americas/sc/extensions/system/application/SCAppConstants.java [hunk 1]
```diff
@@ -9,6 +9,12 @@
...
 
+  /** Filename of jvmrun file */
+  public static final String JVM_RUN_FILENAME = "jvmrun";
+
+  /** Directory jvmrun file is located in */
+  public static final String JVM_RUN_DIRECTORY = "/usr/";
+
   /** File path for jvmrun file */
-  public static final String JVM_RUN_PATH = "/usr/jvmrun";
+  public static final String JVM_RUN_PATH = JVM_RUN_DIRECTORY + JVM_RUN_FILENAME;
```
[the remark this answers is not in the document]
Comment (TomKimsey) [reply src/main/java/com/hms_networks/americas/sc/extensions/system/application/SCAppConstants.java]: changed

File: src/main/java/com/hms_networks/americas/sc/extensions/system/application/SCAppRebootWatchdog.java [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+ * loops can be difficult to access via http, ftp, or vpn.
+ *
+ * @since 1.14.0
+ * @author HMS Networks, MU Americas Solution Center
+ */
+public class SCAppRebootWatchdog {
+
+  /** Path for jvmrun filename when disabled by the watchdog */
+  private static final String DISABLED_JVM_RUN_PATH =
+      SCAppConstants.JVM_RUN_DIRECTORY + "disabled_" + SCAppConstants.JVM_RUN_FILENAME;
```
[the remark this answers is not in the document]
Comment (TomKimsey) [reply src/main/java/com/hms_networks/americas/sc/extensions/system/application/SCAppRebootWatchdog.java]: changed

**Review by alexjhawk — APPROVED** [review]

**Comment by alexjhawk** [comment]
> > > I think this looks good to me.
> > > Also, what do you and @it-hms think about adding automation to help us better update connectors when the extensions library releases are made? I think it would be extremely useful to automatically log an issue on connector repositories that an update needs to be made, otherwise features like this tend to be forgotten and only make it to 1 or 2 connectors.
> > 
> > 
> > Yeah, it makes sense to have automation to log an issue.
> 
> I like the idea too but what would it look like? Would it just be a issue to "Implement features introduced in v1.14.0" or would it be more customizable like "Add Reboot Watchdog functionality".
> 
> Maybe you could create an issue in the extensions lib then utilize a custom tag in the issue like "ConnectorRequiredUpdate" that could be used to replicate that issue across all connectors. Then i guess when all connectors have closed the issue you could close it in the extensions lib.

That is a good idea. I didn't think of a solution like that, but I think it would be more flexible.

My original thought was something where we just take the release version number and changelog, then create an issue such as 
- Update extensions library to v1.14.0
- CHANGELOG: 
  - Change 1
  - Change 2

**Review by it-hms — APPROVED** [review]

**Pull request merged**